targets/xorg

```diff
@@ -68,7 +68,7 @@ if [ "${ARCH#arm}" = "$ARCH" ]; then
 fi
 
 # Catalog relevant and irrelevant video drivers
-fbdev="xserver-xorg-video-fbdev$backport"
+fbdev="openSUSE=xf86-video-fbdev,xserver-xorg-video-fbdev$backport"
```

Comment:
I wonder if it would be cleaner to have each distro set the xorg package name prefix at the top and then just reference `$xorg-video-fbdev` (etc) without needing any distro modifiers